### PR TITLE
[11.0][MIG]product special type

### PR DESCRIPTION
**Pull request by HirenDangar**
product special type
================

### CODE REVIEW

File: product_special_type/models/product.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2012 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class product_template(models.Model):
```
Comment (nikul-serpentcs) [inline product_special_type/models/product.py]:
```suggestion
class ProductTemplate(models.Model):
```

File: product_special_type/models/product.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    _inherit = 'product.template'
+
+	special_type = fields.selection([
```
Comment (nikul-serpentcs) [inline product_special_type/models/product.py]:
```Suggestion
	special_type = fields.Selection([
```

File: product_special_type/models/product.py (hunk 1 above)
Comment (HirenDangar) [reply product_special_type/models/product.py]: **Yes brother I'll make CamelCase.**

**Comment by HirenDangar** [comment]
Hey @Nikul-Chaudhary  Can you please confirm with OCA guy's for my CLA. I already sent.

**Comment by Nikul-Chaudhary** [comment]
> Hey @Nikul-Chaudhary Can you please confirm with OCA guy's for my CLA. I already sent.

Yes Sure

CC @pedrobaeza @jbeficent @OCA/board  can you please help here.

**Comment by pedrobaeza** [comment]
Hi, Hiren, it seems you are covered by Serpent ECLA.

**Comment by HirenDangar** [comment]
But If I want to contribute personally ?
I get this CLA so I can contribute more from my way.

**Comment by rousseldenis** [comment]
FYI, I did this https://github.com/OCA/product-attribute/pull/419

**Comment by pedrobaeza** [comment]
@HirenDangar no problem on that. The registered email is your personal one.

**Comment by rousseldenis** [comment]
> FYI, I did this [OCA/product-attribute#419](https://github.com/OCA/product-attribute/pull/419)

IMHO as this module does not depend on sale module and only on product one, this should be done on 'product-attribute' repo

**Comment by legalsylvain** [comment]
Agree with @rousseldenis

**Comment by HirenDangar** [comment]
> @HirenDangar no problem on that. The registered email is your personal one.

Hello @pedrobaeza ,
But this is my personnel account and I already sent CLA from my personnel email like:- **hirendangar@gmail.com**
Can you please look out here
Github = HirenDangar (https://github.com/HirenDangar)
Name= Hiren Dangar
Still I'm waiting for confirmation.

**Comment by pedrobaeza** [comment]
Everything is OK, so you can continue.

**Comment by rafaelbn** [comment]
As commented:  https://github.com/OCA/sale-workflow/pull/902#issuecomment-513532488 / https://github.com/OCA/sale-workflow/pull/902#issuecomment-513683003 

This PR is in the incorrect repo, the correct one is https://github.com/OCA/product-attribute

The module is https://github.com/OCA/product-attribute/tree/10.0/product_special_type

**Comment by rousseldenis** [comment]
@HirenDangar Closing as wrong repo. Please attend comments.

**Pull request closed without merging**